src/framework/ui/internal/uiactionsregister.h

```diff
@@ -69,6 +73,8 @@ class UiActionsRegister : public IUiActionsRegister, public Injectable, public a
     void updateShortcuts(const actions::ActionCodeList& codes);
     void updateShortcutsAll();
 
+    void updateActions(const UiActionList& actions);
```

Comment:
Not `override` ? Aren't clients supposed to interact with abstract `IUiActionsRegister` ?

Reply:
not override
the reasons are described here (number two)
https://github.com/musescore/MuseScore/pull/24834#issuecomment-2363578645